openid-4-verifiable-presentations-1_0.md

```diff
@@ -283,6 +289,17 @@ The following is a non-normative example of an Authorization Request:
     &nonce=n-0S6_WzA2Mj HTTP/1.1
 ```
 
+The following is a non-normative example with a `request_uri_mode` parameter (including the additional parameters `client_id_scheme` and `client_metadata`): 
+
+```
+  GET /authorize?
```

Comment:
I think the indentation should be removed here.

Reply:
we use the same indentation for all GET request examples. suggest to clean that up in another PR